reaper/reaper/src/main/kotlin/com/emergetools/reaper/ReaperInternal.kt

```diff
@@ -121,6 +121,20 @@ internal object ReaperInternal {
       Log.e(TAG, "Reaper already initialized, ignoring Reaper.init().")
       return
     }
+
+    val isEnabled = context.packageManager.getApplicationInfo(
+      context.packageName,
+      PackageManager.GET_META_DATA
+    ).metaData.getBoolean(MANIFEST_TAG_INSTRUMENTED, false)
+
+    if (!isEnabled) {
+      // Explicitly don't use fatalError to ensure we don't crash other variants
+      Log.w(TAG,
+        "Reaper is not enabled, ensure this variant is specified in the reaper.enabledVariants list in the Emerge gradle plugin configuration block."
```

Comment:
I would keep the link and make it more spesific to the helpful section: https://docs.emergetools.com/docs/reaper-setup-android#configure-the-sdk